tests/virtualization/syscontainer_image_test.pm

```diff
@@ -0,0 +1,128 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: system containers images test setup, installing libvirt-lxc
```

Comment:
That summary is of no use if it's just repeating the same as in the other file(s), please be more specific what this certain module does